packages/@okta/vuepress-site/docs/reference/api/hook-keys/index.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Key Management
+category: management
+excerpt:
+  The Key Management API provides a CRUD interface for registering key
+  endpoints to use with Hooks.
+---
+
+# Key Management API
+
+The Okta key management API provides a CRUD interface for JSON Web Keys (JWK) used with other parts of the application, such as inline hooks. For information on how to create inline hooks, see [Inline hooks management API](/docs/reference/api/inline-hooks/).
+
+## Get started
+
+Explore the key management API:[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/a36f6c129009072f78b5?action=collection%2Fimport)
+
+## Key management operations
+
+The key management API has the following CRUD operations:
+
+* [Create a key](#create-a-key)
+* [Get a key](#get-a-key)
+* [List keys](#list-keys)
+* [Get a public key](#get-a-public-key)
+* [Update a key](#update-a-key)
+* [Delete a key](#delete-a-key)
+
+### Create a key
+
+<ApiOperation method="post" url="/api/v1/hook-keys" />
+
```

Comment:
Creates a key for use with inline hooks

Reply:
I believe they want to expand the use of this API past hooks so were keeping this title generic